invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/api/DepositRecordSerializer.js

```diff
@@ -372,13 +398,13 @@ export class RDMDepositRecordSerializer extends DepositRecordSerializer {
       if ("severity" in e && "severity" in e) {
```

Comment:
this looks like a typo maybe we should change it to 
`if ("severity" in e && "message" in e ) {` ?